src/main/java/com/codecool/dungeoncrawl/data/GameMap.java

```diff
@@ -58,6 +57,16 @@ public Wizard getWizard() {
     public Player getPlayer() {
         return player;
     }
+
+    public Chicken getChicken() {
```

Comment:
You have a field `chicken` but you try and access the chicken from a list of enemies which is great, but not sure why you do have the field. :D